frontend/src/router/routeHandlers.ts

```diff
@@ -30,24 +32,35 @@ import type { RouteLocationNormalized } from 'vue-router';
 
 const ACCESS_RESTRICTED_ERROR = new FamRouteError(
     RouteErrorName.ACCESS_RESTRICTED,
-    "Access restricted"
+    'Access restricted'
 );
 
 // --- beforeEnter Route Handler
 
 const beforeEnterDashboardRoute = async (to: RouteLocationNormalized) => {
     let applicationAdmins;
     let userRolesFetchResult;
+    let delegatedAdmins;
+
+    delegatedAdmins = await asyncWrap(fetchDelegatedAdmins(selectedApplicationId.value))
+    // Requires fetching applications the user administers.
+
+    if (LoginUserState.hasAccess(FAM_ADMIN_ROLE)) {
```

Comment:
Hi Jota, not sure why we fetch delegated admin twice here, maybe you forgot to remove. But we should only fetch delegated admin when selected application is not FAM (which is the else statement below), and user is app admin of the selected application, so not just check if the user has admin role, the user has to be the admin of the selected application

Reply:
thanks, yeah...it should have been removed.